CallWithDoxDialer/DoxDialerCaller.h

```diff
@@ -0,0 +1,17 @@
+//
+//  DoxDialerCaller.h
+//  CallWithDoxDialer
+//
+//  Created on 4/25/17.
+//  Copyright © 2017 Doximity. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface DoxDialerCaller : NSObject
+
++(id _Nonnull)shared;
+
+-(void)dialPhoneNumber:(nonnull NSString *)phoneNumber;
```

Comment:
not sure where we wanted to put documentation, but maybe a good idea to include some documentation here about what type of strings are accepted / will succeed!